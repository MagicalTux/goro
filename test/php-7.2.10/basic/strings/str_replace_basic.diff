     *** Testing str_replace() on basic operations ***
002- string(0) ""
003- string(4) "tbst"
004- string(0) ""
005- int(0)
006- string(1) "q"
007- int(1)
008- string(0) ""
009- int(0)
010- string(%d) "Resource id #%d"
011- int(1)
012- ===DONE===
002+ 
003+ Fatal error: Uncaught Error: str_replace(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php:11
004+ Stack trace:
005+ #0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php(11): str_replace(, , )
006+ #1 {main}
007+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php on line 11
