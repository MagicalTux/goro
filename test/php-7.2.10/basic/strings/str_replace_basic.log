
---- EXPECTED OUTPUT
*** Testing str_replace() on basic operations ***
string(0) ""
string(4) "tbst"
string(0) ""
int(0)
string(1) "q"
int(1)
string(0) ""
int(0)
string(%d) "Resource id #%d"
int(1)
===DONE===
---- ACTUAL OUTPUT
*** Testing str_replace() on basic operations ***

Fatal error: Uncaught Error: str_replace(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php:11
Stack trace:
#0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php(11): str_replace(, , )
#1 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php on line 11
---- FAILED
