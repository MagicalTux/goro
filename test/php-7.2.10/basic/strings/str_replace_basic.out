*** Testing str_replace() on basic operations ***

Fatal error: Uncaught Error: str_replace(): Too few arguments in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php:11
Stack trace:
#0 /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php(11): str_replace(, , )
#1 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/str_replace_basic.php on line 11